### PR TITLE
Fix SSH Session 'cmd', 'pty', 'terminal' types

### PR DESCRIPTION
**Pull request by unbreakabl3**
### Checklist

- [x] I have added relevant error handling and logging messages to help troubleshooting
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation, relevant usage information (if applicable)
- [x] I have updated CHANGELOG.md with a short summary of the changes introduced
- [ ] I have tested against live environment, if applicable
- [ ] I have synced any structure and/or content vRA-NG improvements with vra-ng and ts-vra-ng archetypes (if applicable)
- [ ] I have my changes rebased and squashed to the minimal number of relevant commits. **Notice: don't squash all commits**
- [ ] I have added a descriptive commit message with a short title, including a `Fixed #XXX -` or `Closed #XXX -` prefix to auto-close the issue

### Release Notes
In the existing vRO API SSHSession, `cmd`, `pty`, `terminal` have type `string`. In `o11n-plugin-ssh` it has type `void`.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		"titleBar.inactiveBackground": "#8e9b7899",
 		"titleBar.inactiveForeground": "#15202b99"
 	},
-    "peacock.color": "#8e9b78",
-    "git.alwaysSignOff": true
-}
+	"peacock.color": "#8e9b78",
+	"git.alwaysSignOff": true,
+	"java.compile.nullAnalysis.mode": "disabled"
```
Comment (VenelinBakalov) [inline .vscode/settings.json:17]: @unbreakabl3 just to check, is this change intentional

**Review by VenelinBakalov — APPROVED** [review]

**Review by VenelinBakalov — APPROVED** [review]

**Comment by VenelinBakalov** [comment]
Hi @unbreakabl3 , could you check the PR, there are some merge conflicts appearing that are blocking the merge

**Comment by unbreakabl3** [comment]
> Hi @unbreakabl3 , could you check the PR, there are some merge conflicts appearing that are blocking the merge

Hi @VenelinBakalov .
Sure. I see now. The problem is that two different commits handling two different issues were added to this PR. I will close that PR and will create a two separated, which are addressing two different issues.

**Comment by VenelinBakalov** [comment]
Nice, thanks, the new PR is approved!
